carbon-design-system/carbon
# fix: web-components - specify js extensions for icon imports

Closes #17813 
Closes #17650 

For users importing the `@carbon/web-components` package and using webpack, there is an error appearing - mandating the specification of the extension in the icon imports.

#### Changelog

**Changed**

- add `.js` extensions to all the icon imports in the `@carbon/web-components` component files
- add `.js` extension to the `spread.js` file that is included in all exported icon files

#### Testing / Reviewing
- open example folder provided by adopter - I already added and linked the package tarball with the fix [example-with-fix.zip](https://github.com/user-attachments/files/17463515/example-with-fix.zip)
- ensure when running `npm install & npm run build` there are no errors appearing
